test/ig/IGExporter.IG.test.ts

```diff
@@ -951,7 +941,7 @@ describe('IGExporter', () => {
       patientInstance._instanceMeta.usage = 'Example';
       pkg.instances.push(patientInstance);
 
-      exporter = new IGExporter(pkg, defs, path.resolve(fixtures, 'ig-data'), true);
+      exporter = new IGExporter(pkg, defs, path.resolve(fixtures, 'ig-data'));
```

Comment:
Is this just because we already have our test cases set up to put stuff in `ig-data` and didn't want to go through the trouble of all the changes to move it? I vaguely remember something like that.

Reply:
Yeah I left these because once we provide the path to the `ig-data` folder, the content in that folder has the same shape and content as we would expect in the root tank folder now. I was thinking it would reduce complexity of the changes, but it could be nice to go through all the test fixtures and remove all the `ig-data` folders and move the content up a directory.

@ngfreiter do you have a preference? There are a lot of test fixtures with `ig-data` folders if that influences your thought process in either direction. Also happy to hear @cmoesel's thoughts on this.

Reply:
I'm fine with leaving it, we can pretend it's just a fun little name we use for our tests.